packages/clerk-js/src/core/resources/User.ts

```diff
@@ -80,6 +80,8 @@ export class User extends BaseResource implements UserResource {
   publicMetadata: UserPublicMetadata = {};
   unsafeMetadata: UserUnsafeMetadata = {};
   lastSignInAt: Date | null = null;
+  createOrganizationEnabled: boolean = false;
```

Comment:
Same here with the leaked properties note above

Reply:
❓ Are these named based on the FAPI payload? Another naming that'd make sense to me would be `canCreateOrganization` but I'm fine with `createOrganizationEnabled` as well.

Reply:
Yes, it is based on FAPI. We iterated a lot on naming these parameters. I can't recall why we didn't pick the `canX` proposal that was discussed at some point.

@mzhong9723 @jescalan Do you maybe remember?

@nikosdouvlis FYI [this is the design doc](https://www.notion.so/clerkdev/Allow-our-users-to-delete-themselves-and-their-orgs-94180a172b0e428c9352835274fa8bfe). You can also check the comments section for naming decisions.

Reply:
as far as `canX`, I'm not sure to be honest, I just mirrored the property name from bapi for consistency